storage/rocksdb/ha_rocksdb.cc

```diff
@@ -14085,12 +14108,18 @@ int ha_rocksdb::delete_table(const char *const tablename,
     HA_EXIT_SUCCESS  OK
     other            HA_ERR error code (cannot be SE-specific)
 */
-int ha_rocksdb::rename_table(
-    const char *const from, const char *const to,
-    const dd::Table *from_table_def MY_ATTRIBUTE((__unused__)),
-    dd::Table *to_table_def MY_ATTRIBUTE((__unused__))) {
+int ha_rocksdb::rename_table(const char *const from, const char *const to,
+                             const dd::Table *from_table_def,
+                             dd::Table *to_table_def
+                                 MY_ATTRIBUTE((__unused__))) {
```

Comment:
Since this is now touched, `[[maybe_unused]]`

Reply:
For `to_table_def`, since you are touching that line due to reformatting, I am suggesting to replace `MY_ATTRIBUTE((__unused__))` with `[[maybe_unused]]`